crt_portal/cts_forms/views.py

```diff
@@ -99,8 +99,21 @@ def IndexView(request):
 @login_required
 def ShowView(request, id):
     report = get_object_or_404(Report.objects, id=id)
+    primary_complaint = [choice[1] for choice in PRIMARY_COMPLAINT_CHOICES if choice[0] == report.primary_complaint]
+    crimes = {
+        'physical_harm': False,
+        'trafficking': False
+    }
+
+    for crime in report.hatecrimes_trafficking.all():
+        for choice in HATE_CRIMES_TRAFFICKING_MODEL_CHOICES:
+            if crime.hatecrimes_trafficking_option == choice[1]:
```

Comment:
This feels brittle to me because it is based on the order of the content of each tuple being the same. I'm wondering if in a future / follow up PR, it makes more sense to store a reference to the type of crime in the report, and use that to match against the hate crime description.

For example, in the `protected_class` field, we store `disability`, not the string that choice maps to.

Reply:
We can simplify this a bit. We should be able to see if an object is in a list or set without a loop. 

We might be able to use the option choice name instead of the text. We do that in the section assignment method I can't remember if it only works as a method or if it works outside the class too. 
```
If 'physical_harm' in report.hatecrimes_trafficking.all():
    crimes['physical_harm'] = True
``

Reply:
Ah cool, if we can do this lookup I'd prefer that to looping and matching on the string!